php-sdk-release/Dockerfile

```diff
@@ -5,13 +5,13 @@ RUN apt-get -q update && apt-get -y install zip
 
 WORKDIR /home
 
-ENV PHPDOCUMENTOR_ARCHIVE_FILE=phpDocumentor.phar
+ENV PHP_TOOLS_DIR=/home/launchdarkly/tools/php
+ENV PATH="${PATH}:${PHP_TOOLS_DIR}"
 
-ENV LDTOOLS_DIR=/home/launchdarkly/tools
+RUN mkdir -p ${PHP_TOOLS_DIR}
 
-RUN mkdir -p ${LDTOOLS_DIR}
-
-ADD downloads/${PHPDOCUMENTOR_ARCHIVE_FILE} ${LDTOOLS_DIR}/php/${PHPDOCUMENTOR_ARCHIVE_FILE}
+ADD downloads/composer ${PHP_TOOLS_DIR}/composer
```

Comment:
[Docker best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy) suggest preferring COPY over ADD unless you are using ADD to fetch remote resources or extract a tar file.

Not worth blocking over, but I did want to mention it.